File: Python/interest.py
principle = float(raw_input("Principle: "))
rate = float(raw_input("Interest Rate: "))
n = float(raw_input("Number of times paid per year: "))
time = float(raw_input("Years: "))
print principle * pow(1+(.01*rate/n), (n*time))
raw_input

File: C++_2005/Interest/Interest.cpp
#include <iostream.h>
#include <math.h>

int main()
{
	double principle, rate, n, time;

	cout << "Principle: ";
	cin >> principle;
	cout << "Interest Rate: ";
	cin >> rate;
	cout << "Paid per year: ";
	cin >> n;
	cout << "Years: ";
	cin >> time;

	cout << principle * pow(1+(.01*rate/n), (n*time));

	return 0;
}